api/src/main/java/org/bf2/operator/ManagedKafkaKeys.java

```diff
@@ -61,7 +61,11 @@ public static final class Annotations {
         /**
          * Set of annotations managed by the data plane - expand as needed.
          */
-        public static final Set<String> DATA_PLANE_ANNOTATIONS = Set.of(MASTER_SECRET_DIGEST, PAUSE_RECONCILIATION);
+        public static final Set<String> DATA_PLANE_ANNOTATIONS = Set.of(
+                MASTER_SECRET_DIGEST,
+                KAFKA_UPGRADE_START_TIMESTAMP,
+                KAFKA_UPGRADE_END_TIMESTAMP,
```

Comment:
It's not clear to me how having these annotations missing today does not interfere with the Kafka version upgrade process.

Reply:
It sure seems like it could be a problem.  Should this be separated into a different pr for immediate inclusion?

Reply:
After discussing in the standup, I think we're safe to leave this as-is without an immediate/separate fix.